src/options.js

```diff
@@ -183,9 +183,13 @@ function populateObject (object, defaultValues) {
 }
 
 function getTests (options) {
-    var tests = readJSON(options.tests, defaults.tests);
-
-    if (!options.results && !options.resultIds) {
+    var tests = options.tests;
+	
+	if(! Array.isArray(tests)) {
```

Comment:
Indentation in this project is four spaces in this project, not tabs. I realise that differs to some other Nature projects; are you using the DetectIndent vim plugin? It makes life easier.